### PR TITLE
Tuning cycle explanations

### PR DESCRIPTION
**Pull request by cderb**

### CODE REVIEW

File: doc/TuningCycle.md [hunk 1]
```diff
@@ -26,3 +26,80 @@ please ask for them. The Tuna team does not provide these.
...
+solver_applicability table with our applicable solvers for each configuration for the session.
+
+./go_fish.py --local_machine --update_applicability --session_id 1
+--session_id - tuning session id
+
+### Load Jobs
+Time to create the jobs for the tuning session. We'll need to specify the session id, the configs that
+should be tuned, the fin_step to be executed. Jobs should have a compile and an eval fin step pair.
+
+fin steps include miopen_perf_compile, miopen_perf_eval, miopen_find_compile, miopen_find_eval
```
Comment (alexandraBara) [inline doc/TuningCycle.md]:
```suggestion
Fin steps include: miopen_perf_compile, miopen_perf_eval, miopen_find_compile and miopen_find_eval.
```

File: doc/TuningCycle.md [hunk 2]
```diff
@@ -26,3 +26,80 @@ please ask for them. The Tuna team does not provide these.
...
+
+./go_fish.py --local_machine --init_session -l reason
+--init_session - create a session entry
+-l             - reference text description
+
+### Create Applicability
+Each network configuration has a set of applicable solvers. In this step we update the
+solver_applicability table with our applicable solvers for each configuration for the session.
+
+./go_fish.py --local_machine --update_applicability --session_id 1
```
Comment (alexandraBara) [inline doc/TuningCycle.md]: Add every command in quotations so it shows up nicely formatted and distinct from text.

```suggestion
```./go_fish.py --local_machine --update_applicability --session_id 1```
```

File: doc/TuningCycle.md [hunk 3]
```diff
@@ -26,3 +26,80 @@ please ask for them. The Tuna team does not provide these.
...
+
+./load_job.py --session_id 1 -t resnet50 --fin_steps miopen_perf_compile,miopen_perf_eval -o -l reason
+--session_id - tuning session id
+-t           - config tag
+--fin_steps  - operations to be performed by fin (tuning handle into miopen)
+-o           - only_applicable, will create a job for each applicable solver
+-l           - reference text description
+
+### Compile Step
+Finally time to start executing the jobs. To compile the jobs supply the session id
```
Comment (alexandraBara) [inline doc/TuningCycle.md:81]:
```suggestion
Once prerequisites are set, tuning can begin. To compile the jobs supply the session id
```

File: doc/TuningCycle.md [hunk 4]
```diff
@@ -26,3 +26,80 @@ please ask for them. The Tuna team does not provide these.
...
+along with the fin_step for the job.
+
+./go_fish.py --local_machine --session_id 1 --fin_steps miopen_perf_compile
+--local_machine - run directly on this machine
+--session_id    - tuning session id
+--fin_steps     - execute this operation
+
+### Evaluation Step
+The job is ready for evaluation after the compile step, this command is similar to the previous
```
Comment (alexandraBara) [inline doc/TuningCycle.md:90]:
```suggestion
Once compile jobs have been started, the evaluation step can begin. This command is similar to the previous
```

File: doc/TuningCycle.md [hunk 5]
```diff
@@ -26,3 +26,80 @@ please ask for them. The Tuna team does not provide these.
...
+
+./go_fish.py --local_machine --session_id 1 --fin_steps miopen_perf_eval
+--local_machine - run directly on this machine
+--session_id    - tuning session id
+--fin_steps     - execute this operation
+
+### Database Export
+To export the results the export_db.py script can be run with options
```
Comment (alexandraBara) [inline doc/TuningCycle.md:98]: Add some more details that these results are exported into a .db or .fdb txt format and contains the information from the central tuning DB.

**Review by alexandraBara — CHANGES_REQUESTED** [review]

File: doc/TuningCycle.md [hunk 6]
```diff
@@ -26,3 +26,80 @@ please ask for them. The Tuna team does not provide these.
 phase of all the recurrent configurations takes up to a few days. There are many configurations
 used for each network and one should try and use as many machines as possible to speed up
 the tuning part. The current configurations we tune for are:
+
+## Tuning Steps
```
[the remark this answers is not in the document]
Comment (cderb) [reply doc/TuningCycle.md:30]: This should do quite well.

**Review by alexandraBara — APPROVED** [review]

**Review by alexandraBara — APPROVED** [review]
LGTM

**Review by JehandadKhan — APPROVED** [review]

**Pull request merged**